packages/migrations/src/schema/create_tables.sql

```diff
@@ -76,7 +76,7 @@ CREATE TABLE public.certification_request (
     "createdAt" timestamp with time zone DEFAULT now() NOT NULL,
     "updatedAt" timestamp with time zone DEFAULT now() NOT NULL,
     id integer NOT NULL,
-    energy integer NOT NULL,
+    energy character varying NOT NULL,
```

Comment:
This should go to the migrations stuff. Which is still WIP #851

Reply:
It's needed for this to work. Should I way for the WIP PR to be done?

Reply:
Yes, I think so, let's keep this one open, the #851 should be done soon.